joomla/joomla-cms
# [com_tags] Add search tooltip

#### Summary of Changes

Simple PR to add the search tooltip and element label (accessibility) in com_tags.
#### Testing Instructions

Code review, or:
1. Apply patch
2. Go to Components -> Tags, check the tooltip in the search field.
